app/views/projects/_form.html.erb

```diff
@@ -2,6 +2,25 @@
 
   <%= submit_button(form: f, button: button, center: true) %>
 
+
+  # TODO improve date range area styling. Maybe a panel block. See show template.
+  <%= tag.div(
+    tag.p("*#{:form_project_date_range.t}*".t) +
+    tag.p(:form_project_dates_explain.t)
+  ) %>
+  <div class="container-text">
+    <div>
+      <%= f.label(:start_date, "#{:form_projects_start_date.t}: ") %>
+      <%= f.date_select(:start_date,
+                        { class: "form-control mb-2" }) %>
+    </div>
+    <div>
+      <%= f.label(:end_date, "#{:form_projects_end_date.t}: ") %>
+      <%= f.date_select(:end_date,
+                        { class: "form-control mb-2" }) %>
+    </div>
+  </div>
```

Comment:
Quick & Dirty.
I need to figure out how to style this with @nimmolo's helpers and css classes.

Reply:
Looks pretty good. I think your `<div>`s maybe need `class="form-group form-inline"` (or possibly just around the `date_select`'s three fields? Currently my styling reference is the [Bootstrap 3 form](https://getbootstrap.com/docs/3.4/css/#forms) docs.

I have a `select_with_label` but haven't written a `date_select_with_label` yet.

Reply:
I will leave this for later, as Bootstrap wants a lot of detail that I don't yet understand.